templates/java/org/prism/Loader.java.erb

```diff
@@ -284,6 +284,11 @@ public class Loader {
         return x;
     }
 
+    private int loadVarSInt() {
+        int x = loadVarUInt();
+        return (x >> 1) ^ (-(x & 1));
```

Comment:
Could you use `>>>` like https://github.com/protocolbuffers/protobuf/blob/main/java/core/src/main/java/com/google/protobuf/CodedInputStream.java#L508-L510?
I would also add a `// From URL` comment with that like above